wgsl/index.bs

```diff
@@ -2082,8 +2082,20 @@ shift_expression
   | shift_expression SHIFT_RIGHT additive_expression
         OpShiftRightLogical or OpShiftRightArithmetic
 
-relational_expression
+and_expression
   : shift_expression
+  | and_expression AND equality_expression
+
+exclusive_or_expression
+  : and_expression
+  | exclusive_or_expression XOR and_expression
+
+inclusive_or_expression
+  : exclusive_or_expression
+  | inclusive_or_expression OR exclusive_or_expression
+
+relational_expression
+  : inclusive_or_expression
   | relational_expression LESS_THAN shift_expression
```

Comment:
This needs to be updated as well

Reply:
@dj2 Does the latest commit address concerns?